app/js/view/main_view.js

```diff
@@ -14,16 +18,27 @@ export default class MainView extends View {
           window.dispatchEvent(new CustomEvent('request-activity-finish'));
         }
       });
+    } else {
+      var uploadBtn = document.getElementById('upload-link');
```

Comment:
MainView does construct the upload-link. See https://github.com/fxos/directory/pull/25/files#diff-9d12d5d1f083436d44649f99f776fbc6R38